test/testdata/dsl/t_struct_none.rb.dsl-tree.exp

```diff
@@ -0,0 +1,17 @@
+class <emptyTree><<C <root>>> < ()
+  class <emptyTree>::<C None><<C <todo sym>>> < (<emptyTree>::<C T>::<C Struct>)
+    ::T::Sig::WithoutRuntime.sig() do ||
+      <self>.params({}).void()
```

Comment:
I'm a little surprised to see this this isn't a type error:

```ruby
class A
  extend T::Sig

  sig {params().void}
  def bar
  end
end
```

^ because this is:

https://sorbet.run/#%23%20typed%3A%20true%0Aclass%20A%0A%20%20extend%20T%3A%3ASig%0A%0A%20%20sig%20%7Bparams().void%7D%0A%20%20def%20bar%0A%20%20end%0Aend%0A

Reply:
In this degenerate case, we pass an empty hash to `params` which seems to make it OK. (Trying it on sorbet.run makes it fine there too.)